daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SError.scala

```diff
@@ -41,13 +41,20 @@ object SError {
     */
   sealed abstract class SErrorDamlException extends SError
 
+  /** Unhandled exceptions */
+  final case class DamlEUnhandledException(message: String) extends SErrorDamlException {
```

Comment:
This is a bit more involved than it seems. Because the payload values would have to be converted back from speedy-values
to normal LLF values. I've marked this as a TODO.